portals/application/configs/klear/model/Users.yaml

```diff
@@ -180,22 +180,19 @@ production:
       trim: both
       maxLength: 65
     maxCalls:
-      title: _('Max call')
+      title: _('Max calls')
       type: number
-      defaultValue: 2
+      defaultValue: 1
```

Comment:
This value is never show, as the creation windows hides maxCalls and sets the value to database default.

I will change it to zero though, in case it is shown in the future :)

Thanks.